### ISSUE TITLE
[Bug]: SWWW Grayed wallpapers & no wallpaper @ login

### ISSUE THREAD

**Issue by kRHYME7** [prasanthrangan/hyprdots#1204]
## Bug Report

Let me compile the issue to avoid dupes. 

#### ISSUES

Please scan this issues before you open another duplicates😁

+  https://github.com/prasanthrangan/hyprdots/issues/1165
+ https://github.com/prasanthrangan/hyprdots/issues/1180

+ https://github.com/LGFae/swww/issues/233  << [UPSTREAM ISSUE]

#### WORKAROUNDS

Here are the ways to fix it. 
basically, we just need to remove the old socket

**Fix Grayed wallpaper** 
https://github.com/LGFae/swww/issues/248#issuecomment-2022795829

``` swww-daemon --format xrgb ``` should be enough

> [!Note]
> clearing cache is useless according to the dev

If you happen to have the error below when you run the above command, it's possible you did not read the thread above.
``` Error: "couldn't bind socket: Address already in use (os error 98)" ```
Basically we need to remove the existing socket  ```  rm  $XDG_RUNTIME_DIR/swww.socket ``` then rerun again. It should work. 
Now, how can you fix it across reboots? Do the workarounds below.

**Wallpaper not loading** 

+  Install the upstream package ``` yay -Sy swww-git ``` 

+   If you don't want to install the devel package,  use this part https://github.com/prasanthrangan/hyprdots/blob/537594a49db2bf9a6e64394e2c685d4d7c9dd4fd/Configs/.config/hyprdots/scripts/swwwallpaper.sh#L140C1-L160C1 to your  ``` ~/.config/hyprdots/scripts/swwwallpaper.sh ``` 

+ This should work too 
```
swww query
if [ $? -ne 0 ] ; then
[ -e $XDG_RUNTIME_DIR/swww.socket ] && rm  $XDG_RUNTIME_DIR/swww.socket
    swww-daemon --format xrgb &
fi
```

**Comment by Mauzy0x** [comment]
Modifying the code in `~/.config/hypr/scripts/swwwallpaper.sh` as described in that link fixed this issue for me. 
Thank you for your post!

**Comment by goheakan** [comment]
Without to do any modification in the config files, I was installing some basics packages, did a reboot and here comes the "black wallpaper".
I modified also the code in `~/.config/hypr/scripts/swwwallpaper.sh` as you described and it fixed this issue for me too.
Thank you @kRHYME7

**Comment by Darshil-154** [comment]
i also faced same issue

**Comment by iEscapedVim** [comment]
> 😭 Why everyone think running a command in sudo fixes everything 😭

idk I thought it was a permission issue or sth? 😅😅😅

**Comment by kRHYME7** [comment]
you should modify your   ``` ~/.config/hyprdots/scripts/swwwallpaper.sh ```   for now. 
open editor then G to the bottom  you'll find this part ``` # check swww daemon and set wall ``` make it same as mine. 
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/6f4c8992-d8fe-4812-a16b-5b8d6aa9b75b)

```
swww query
if [ $? -ne 0 ] ; then
[ -e $XDG_RUNTIME_DIR/swww.socket ] && rm  $XDG_RUNTIME_DIR/swww.socket
    swww-daemon --format xrgb &
fi

Wall_Set
```

So as I've said this is already fixed on the swww-git so you can just install the -git version reboot and will be fine. 
but compiling the -git takes time so as work around we will remove the conflicting socket ourselves hence this part 
``` [ -e $XDG_RUNTIME_DIR/swww.socket ] && rm  $XDG_RUNTIME_DIR/swww.socket ```

@iEscapedVim

**Comment by iEscapedVim** [comment]
> So as I've said this is already fixed on the swww-git so you can just install the -git version reboot and will be fine. but compiling the -git takes time so as work around we will remove the conflicting socket ourselves hence this part `[ -e $XDG_RUNTIME_DIR/swww.socket ] && rm $XDG_RUNTIME_DIR/swww.socket`

I installed it already after reboot the issue is still there

![image](https://github.com/prasanthrangan/hyprdots/assets/32537564/6419de43-e562-47bb-ab09-d5d8146dc9ad)

**Comment by kRHYME7** [comment]
output for of ```  tail 10 ~/.config/hyprdots/scripts/swwwallpaper.sh ``` ?

**Comment by iEscapedVim** [comment]
> tail 10 ~/.config/hyprdots/scripts/swwwallpaper.sh

```❯ tail 10 ~/.config/hyprdots/scripts/swwwallpaper.sh
tail: cannot open '10' for reading: No such file or directory
==> /home/zero/.config/hyprdots/scripts/swwwallpaper.sh <==

# check swww daemon and set wall

swww query
 if [ $? -eq 1 ] ; then
    swww init
fi

Wall_Set
```

**Comment by kRHYME7** [comment]
swww-git will only fix this error ``` Error: "couldn't bind socket: Address already in use (os error 98 ``` 

the graying is another issue unsolve upstream but there is a workaround :
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/32a52986-54e2-48bc-8455-1ea904b8e0a4)


change that part to something like this 



Okay you can continue reading the above thread, I think instructions are sufficient already.

**Comment by iEscapedVim** [comment]
Oh I see when you said 

> So as I've said this is already fixed on the swww-git so you can just install the -git version reboot and will be fine.


I thought, I either had to install that or do the work around.

anyway it's fixed.. Thanks
![image](https://github.com/prasanthrangan/hyprdots/assets/32537564/61e58a14-25f6-482c-8ee4-81a3a11ce6d6)

**Comment by dbt0815** [comment]
so now this workaround doesn't even work anymore because there is no hyper dots folder anymore when I set it all new up yesterday. holy fuck it worked like a charm for me as well as other things a week ago. now not even Autostart commands work.
